src/wp-includes/script-loader.php

```diff
@@ -2336,6 +2336,35 @@ function wp_enqueue_global_styles() {
 	wp_enqueue_style( 'global-styles' );
 }
 
+/**
+ * Render the SVG filters supplied by theme.json.
+ *
+ * Note that this doesn't render the per-block user-defined
+ * filters which are handled by wp_render_duotone_support,
+ * but it should be rendered before the filtered content
+ * in the body to satisfy Safari's rendering quirks.
+ *
+ * @since 5.9.0
+ */
+function wp_global_styles_render_svg_filters() {
+	/*
+	 * When calling via the in_admin_header action, we only want to render the
+	 * SVGs on the post editor page.
+	 */
+	global $pagenow;
+	if (
+		is_admin() &&
+		( 'post.php' !== $pagenow || 'edit' !== $_GET['action'] )
```

Comment:
We can also use `is_admin() && get_current_screen()->is_block_editor()` it should work for the site editor as well.

Reply:
The site editor doesn't need the SVGs because it's rendering the editor in an iframe that doesn't have access to the filters rendered in the page. 

https://github.com/WordPress/gutenberg/pull/37727 is open to render the SVGs via JavaScript into that iframe.

Reply:
Even so, I think `get_current_screen()->is_block_editor()` is still a better way of checking, so I've updated it.